### DISCUSSION TITLE
Presenting fpm at FortranCon 2021

### DISCUSSION THREAD

**Discussion by awvwgk** [fortran-lang/fpm#515] [Ideas]
For FortranCon 2021 we want to present the core projects of the Fortran-lang community again. Here I want to organize the presentation(s) for the fpm project. But first, we are looking for a volunteer who wants to lead the presentation of fpm and give the talk at FortranCon.

Also, for our GSoC student @kubajj working on fpm, we are planning to organize a separate talk as well.

---

You can checkout the presentation of fpm from last year held by @everythingfunctional  [here](https://youtu.be/gO76uWUtKvE)

FortranCon 2021 announcement: https://fortran-lang.discourse.group/t/1485

**Comment by awvwgk** [comment]
I'm myself interested in presenting fpm at FortranCon, but I don't want to take over this talk without given everybody the chance to volunteer here first.

**Comment by everythingfunctional** [comment]
I'd be happy to give the talk, as I did last year. I'm just not quite sure what I'd focus on. I could give the same talk I gave last year, or try and focus on improvements since last year. If anybody else has a good idea and wants to do it, I'd be okay with that too. I'm open to ideas.

**Comment by milancurcic** [comment]
I think both @awvwgk and @everythingfunctional would give a great talk. I'd give preference to @awvwgk for the sake of rotating fpm speakers, and for an early career opportunity.

I suggest the talk be quite different from last year. Here's how I imagine the structure:

1. The what and why of fpm
2. The why and how of Fortran implementation of fpm
3. New features since last year (auto-discovery, linking with libs, external modules, support for examples, plugins, etc.). Here you can give a hook to @kubajj's talk on compiler flags.
4. Planned features

So points 2 and 3 are the new stuff, and could be the meat of the talk.

**Reply by awvwgk** [reply]
All right, then I will take the lead on the fpm presentation. I'll start working on an abstract / talk and share it at the [talks](https://github.com/fortran-lang/talks) repository.

**Reply by LKedward** [reply]
Thanks Sebastian! Please let me know if you would like me to contribute any content/slides.
